xbmc/platform/android/network/NetworkAndroid.cpp

```diff
@@ -234,10 +233,16 @@ std::unique_ptr<CNetworkBase> CNetworkBase::GetNetwork()
   return std::make_unique<CNetworkAndroid>();
 }
 
-CNetworkAndroid::CNetworkAndroid()
- : CNetworkBase()
+CNetworkAndroid::CNetworkAndroid() : CNetworkBase(), CJNIXBMCConnectivityManagerNetworkCallback()
 {
-  RetrieveInterfaces();
+  if (CJNIBase::GetSDKVersion() >= 24)
+  {
+    CJNIConnectivityManager connectivityManager(
```

Comment:
const?

Reply:
It cannot be `const` since it has member functions that are not declared as `const`